AppBuilder/ABFactory.js

```diff
@@ -854,7 +854,7 @@ class ABFactory extends ABFactoryCore {
          console.error(message);
       }
 
-      if (rest && rest.length > 0) {
+      if (rest.length > 0) {
```

Comment:
isn't the point of adding a `rest && rest.length` check to ensure `rest` is a valid value that we can then eval `.length` from?  

Like is it possible `rest` might be `undefined`, `null`,  or some other value that is `falsey`?  If so, doing a `rest.length` is a crash waiting to happen.